src/effective-dart/usage.md

```diff
@@ -380,7 +386,7 @@ class UploadException {
 ```
 
 Assigning to a local variable can be cleaner and safer than using `!` every
-place the field or top-level variable is used:
+place the value is used:
```

Comment:
I understand why you made this change, but now the sentence reads slightly awkward with how close "using" and "used" are. 

Maybe we could be more specific? Since technically you can use the still nullable version without adding `!` as well. Plus it would avoid the extra "use" :)

```suggestion
time you need to treat the value as non-null:
```